examples/model_selection/plot_precision_recall.py

```diff
@@ -37,11 +37,17 @@
 
 :math:`R = \\frac{T_p}{T_p + F_n}`
 
-These quantities are also related to the (:math:`F_1`) score, which is defined
-as the harmonic mean of precision and recall.
+These quantities are also related to the :math:`F_1` score, which is usually
+defined as the harmonic mean of precision and recall.
 
 :math:`F1 = 2\\frac{P \\times R}{P+R}`
 
+However, for practical purposes, Scikit-Learn uses a definition of :math:`F_1`
+that is equivalent except that it is defined (as 0) when precision and recall
```

Comment:
```suggestion
that is equivalent except that it is defined as 0 when precision and recall
```

Reply:
In fact, let's just make this less verbose, and say "F1 is defined as 0 when there are no true positives."

Reply:
To be clear: this level of detail feels very reactive to the situation at hand, rather than being relevant information to ongoing readers (unaware of the regression etc).

Reply:
> F1 is defined as 0 when there are no true positives

^this is not true. None of our formulae is defined when there are only true negatives. That is the case when `zero_division` is supposed to define the behavior of `f1_score`